src/App/Fossa/Config/ListTargets.hs

```diff
@@ -69,10 +69,12 @@ mergeOpts cfgfile _envvars ListTargetsCliOpts{..} = do
 
 collectExperimental :: Maybe ConfigFile -> ExperimentalAnalyzeConfig
 collectExperimental maybeCfg =
-  ExperimentalAnalyzeConfig $
-    fmap
-      gradleConfigsOnly
-      (maybeCfg >>= configExperimental >>= gradle)
+  ExperimentalAnalyzeConfig
+    ( fmap
+        gradleConfigsOnly
+        (maybeCfg >>= configExperimental >>= gradle)
+    )
+    False -- This should be ok because its discovery should not work differently than the old Go modules tactic.
```

Comment:
No, the new tactic only works on things that have already been discovered, like a `go.mod` file. Do you think `True` is the better option?

Reply:
Not blocking on this, but personally I would be a fan of making this `True` so that if we _did_ have any issues with the new strategy in discovery, we find them ahead of time.

Reply:
I can, but I don't think we would actually find them ahead of time. This code is a new tactic (I've been loose with tactic vs. strategy in discussing this, I apologize) inside of the existing Go modules strategy, so the discovery code is exactly the same as what's used when for the old `go mod graph` tactic . 

Essentially the strategy is handed a project with a `go.mod` to investigate and *then* it makes the decision of old vs. new tactic.

Reply:
Hi Jess, I went ahead and created an enum for this.